docs/source/tutorials/playing-command.rst

```diff
@@ -0,0 +1,150 @@
+=========================
+Playing with your command
+=========================
+
+Now that you have started :doc:`your first plugin <first-plugin>`, maybe you
+want to write a more interesting command than the basic ``.hello`` one. Not
+that there is anything wrong with that command! The emoticons plugin is
+composed of commands like this one::
+
+    @plugin.command('shrug')
+    @plugin.action_command('shrugs')
+    def shrug(bot, trigger):
+        bot.say('¯\\_(ツ)_/¯')
+
+Which is one of the maintainers' favorite commands to use. However, let's see
+if we can do something a bit more *complex* than that.
+
+
+Greeting a user by their name
+=============================
+
+Have you noticed that a :ref:`plugin callable <Plugin callables>` takes **two
+arguments?** The first one is the ``bot``, i.e. an instance of Sopel (wrapped
+for your convenience) that you can use to
+:doc:`interact with the bot </plugin/bot>`.
+
+In the previous tutorial, we used ``bot.reply``, which is convenient but not
+always what you want. Maybe you want the bot to say this::
+
+    <YourNick> .hello
+    <Sopel> Hello YourNick, have a nice day!
+
+For that, you need **the second argument**: the ``trigger``. It is an object
+with all the information you need about the message that triggered the
+callable, such as the **message** itself, the **channel**, the type of message,
+etc. and what we need for now is the
+:attr:`trigger.nick <sopel.trigger.Trigger.nick>` attribute::
+
+    from sopel import plugin
+
+    @plugin.command('hello')
+    def hello(bot, trigger):
+        """Say Hello <user>, have a nice day!"""
+        bot.say(f'Hello {trigger.nick}, have a nice day!')
+
+.. important::
+
+    If you want to test this with your bot, and your bot is already running,
+    you need to restart the bot so it will load the new version of your plugin.
```

Comment:
Nit: you don't _need_ to, but we don't _have_ to let them know there's any other way to do it :stuck_out_tongue_winking_eye: 

```suggestion
    restart the bot so it will load the new version of your plugin.
```

Reply:
Counter-proposal: Downgrade to "you should" rather than completely removing the prescription.

```suggestion
    you should restart the bot so it will load the new version of your plugin.
```

Reply:
I prefer the more direct approach to avoid a "technical" lie for the greater good.